client/src/components/overview/StarRating.jsx

```diff
@@ -0,0 +1,18 @@
+import StarIcon from '../../../images/star.svg';
+import { roundToNearestQuarter } from '../../../../helpers/helpers.js';
+
+export default function(props) {
+  const rating = roundToNearestQuarter(props.rating);
+  const percentage = Math.round((rating / 5) * 100);
+  return (
+    <div className="star-rating">
```

Comment:
this five empty star render from line 9- line 13 seems hard code and not 'DRY', is there any way to be more elegant?

Reply:
Yes, good point, I could probably do this with an array loop or something